package.json

```diff
@@ -14,6 +14,8 @@
   "homepage": "http://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
+    "budo": "budo",
```

Comment:
This is how I first started playing around with `budo`. We can leave it in or not.